mxfusion/models/factor_graph.py

```diff
@@ -461,36 +467,40 @@ def reconcile_graphs(current_graphs, primary_previous_graph, secondary_previous_
 
         :rtype: {previous ModelComponent : current ModelComponent}
         """
+
```

Comment:
The variable names are a bit confusing. Please change to more self-explanatory names or add some comments about the role of these variables.